### ISSUE TITLE
Resurface removed `RTP17c` subclauses for our future selves

### ISSUE THREAD

**Issue by QuintinWillison** [ably/specification#122]
In response to [this comment](https://github.com/ably/specification/pull/120#discussion_r1039463870) from @lawrence-forooghian.

**Comment by mattheworiordan** [comment]
I believe this belongs to the features repo as this describes the behaviour when making changes that removes subclauses from the spec. Why features and specification are different repos is a [separate discussion that needs addressing](https://ably.atlassian.net/wiki/spaces/SDK/pages/2658828328/How+we+track+library+features?focusedCommentId=2679636057)

**Comment by lawrence-forooghian** [comment]
Not sure I understand, @mattheworiordan — this issue is related to the [guidance that this repo contains for how to remove a spec point](https://github.com/ably/specification/blob/9f04965de17fd0b31f26ce01aec76caee66cc0ef/CONTRIBUTING.md?plain=1#L44-L49). Why do you that this issue belongs in a different repository?

**Comment by mattheworiordan** [comment]
Agreed, I misunderstood what the features repo is for vs this repo.
